setup.py

```diff
@@ -46,6 +46,10 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.6',
```

Comment:
Have we ever tested on Python 2.6? If not, I think we should remove it from `setup.py`.